python/py-libpysal/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-libpysal
+version             4.3.0
+platforms           darwin
```

Comment:
the package does not install any architecture-dependent files so please add `supported_archs noarch` here

Reply:
Thanks done.